src/openms/source/CHEMISTRY/ModificationsDB.cpp

```diff
@@ -50,10 +50,16 @@ using namespace std;
 
 namespace OpenMS
 {
-  ModificationsDB::ModificationsDB()
+  ModificationsDB::ModificationsDB(OpenMS::String unimod_file, OpenMS::String psimod_file)
   {
-    readFromUnimodXMLFile("CHEMISTRY/unimod.xml");
-    readFromOBOFile("CHEMISTRY/PSI-MOD.obo");
+    if (!unimod_file.empty())
```

Comment:
For the application I had in mind, the idea would have been to potentially only modify one of the two files and leaving the other file empty.